build.gradle

```diff
@@ -423,6 +421,13 @@ subprojects {
       ]
     }
 
+    if ( !versions.scala.startsWith('2.13.0-') ) {
```

Comment:
do we think these are permanent? if they potentially aren't, i think it's better to leave in the default list (and git history) and remove based on the opposite conditional here.

Reply:
hmm, my point still holds but I see now we also use this pattern above. In both cases I guess part of the problem is also that we don't have good version comparisons, just strings -- it's unfortunate that the previous one couldn't be `> 2.11` (and same here but opposite, `<= 2.12`).

Reply:
Correct, my intention was to change as little as possible and hence I included few lines related to 2.13 **only**. Maybe it is possible to come up with a clean solution using `> 2.11` and/or `<= 2.12`, I will have to experiment.

Reply:
> do we think these are permanent? if they potentially aren't, i think it's better to leave in the default list (and git history) and remove based on the opposite conditional here.

Done. I decided for a `if ( versions.baseScala.startsWith('2.13') )` (at least for now).